curriculum/challenges/english/01-responsive-web-design/learn-more-about-css-pseudo-selectors-by-building-a-balance-sheet/step-048.md

```diff
@@ -0,0 +1,187 @@
+---
+id: 619410f7040d427d9c782db7
+title: Step 48
+challengeType: 0
+dashedName: step-48
+---
+
+# --description--
+
+You may have noticed that this affected your `freeCodeCamp` text as well, because it is a direct child of a `div` element. Give your `#header h2` selector a `background` property set to `transparent` to fix this.
+
+# --hints--
+
+Your `#header h2` selector should have a `background` property set to `transparent`.
+
+```js
+assert(new __helpers.CSSHelp(document).getStyle('#header h2')?.background === 'transparent');
```

Comment:
```suggestion
assert(new __helpers.CSSHelp(document).getStyle('#header h2')?.backgroundColor === 'transparent');
```

`transparent` wasn't being added as the `background` in firefox - it was only added as the `backgroundColor`. It was added to both `background` and `backgroundColor` in Chrome and Safari so this should work